.python-version

```diff
@@ -0,0 +1 @@
+3.9
```

Comment:
I think this will cause uv to _always_ use 3.9 for this project?

Reply:
Correct ... the idea being that you're dev-ing on the lowest supported version?

Reply:
Oh I don't usually do that; I just ensure that CI passes on the lowest version. 

I like to use some of the newer Python language features while hacking around interactively on a feature, and then only in the last stage of that process when I "productionize" the code, do I ensure it runs on the lowest target version.